data/presets/amenity/community_centre/lgbtq.json

```diff
@@ -21,5 +28,8 @@
         "lgbtq": "primary",
         "community_centre:for": "lgbtq"
     },
+    "reference": {
+        "key": "community_centre:for"
+    },
```

Comment:
It looks like this is not ideal, yet.

Looking at this example https://pr-1200--ideditor-presets-preview.netlify.app/id/dist/#background=nrw_vdop_wms&disable_features=boundaries&id=n1791368627&map=20.00/52.47851/13.42123 the help text does not fit well

<img width="373" alt="image" src="https://github.com/user-attachments/assets/24572d5a-6c14-4a80-ae93-7e29031a7753">

Overpass query: https://overpass-turbo.eu/s/1OL3

I did not research in the Wiki but I assume there should be a better page that describes this preset? Or, if not, lets remove the reference and fall back to the main key.

Reply:
Thanks, you're right. I deleted the reference because the wiki page of amenity=community_centre should be the right one for this preset.

Reply:
Lets tackle this in a separate issue. Maybe there is some wiki page – or one can be created – that is more clear on this because https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcommunity_centre does not have any search terms of "lgb"… 

I agree that the "no reference" case is OK here, because it is unrelated.

Reply:
It is now tracked in https://github.com/openstreetmap/id-tagging-schema/issues/1303